Woof/Woof/View/SitterFlow/SitterProfileView/EditSitterInformationView.swift

```diff
@@ -45,6 +46,9 @@ struct EditSitterInformationView: View {
                 .keyboardType(.decimalPad)
             TextEditorWithPlaceholder(text: $bio, placeholderText: bioInformationPlaceholderText)
                 .cornerRadius(AppStyle.UIElementConstant.cornerRadius)
+            if name.isEmpty || phone.isEmpty || city.isEmpty || pricePerHour.isEmpty {
+                Text(mandatoryText)
+            }
```

Comment:
Why do we need this? We have already had this info in the profile view.
![Simulator Screenshot - iPhone 15 - 2024-03-14 at 11 04 16](https://github.com/IronFoundation/iOS-Woof/assets/32869814/b0961f83-e3cf-4e69-a81e-57083fc1f68a)

Reply:
Fixed with 5e32277